org.zulip.Zulip.appdata.xml

```diff
@@ -21,7 +21,8 @@
   <developer_name>Kandra Labs, Inc.</developer_name>
   <update_contact>support@zulipchat.com</update_contact>
   <releases>
-    <release version="4.0.3" date="2020-02-28" />
+    <release version="5.0.0" date="2020-04-01"/>
+    <release version="4.0.3" date="2020-02-28"/>
```

Comment:
Just leave the latest release entry. Exhaustive listing of past releases (and accompanying description) is not worth it if not managed by upstream, in my experience.